packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewer/PdfViewer.tsx

```diff
@@ -21,6 +32,16 @@ interface Props {
    */
   scale: number;
 
+  /**
+   * Render text layer
+   */
+  showTextLayer?: boolean;
```

Comment:
Is there a reason for making this an option? Why not _always_ show the text layer when rendering a PDF?

Reply:
I think that we reuse this component to render thumbnail image or tile.

Reply:
I'm worried that this would take up a lot of resources, if we need to create many "thumbnails". We'll have to test this out and make sure we don't overwhelm the browser.

SDU Annotator has a server-side process to create the thumbnails. I don't believe we will be able to make use of that, but a similar approach may be needed here.

Reply:
OK. Let's think about it when we implement thumbnails. With that, our usage is only to display a (full-page) document and I drop the option.